api/capi/src/nnstreamer-capi-util.c

```diff
@@ -1017,16 +1023,30 @@ ml_validate_model_file (char **model, unsigned int num_models,
     ml_nnfw_type_e * nnfw)
 {
   int status = ML_ERROR_NONE;
+  gboolean is_dir = FALSE;
   gchar *pos;
   gchar **file_ext;
   guint i;
 
-  if ((status = _ml_validate_model_file (model, num_models)) != ML_ERROR_NONE)
-    return status;
-
   if (!nnfw)
     return ML_ERROR_INVALID_PARAMETER;
 
+  status = _ml_validate_model_file (model, num_models, &is_dir);
+  if (status != ML_ERROR_NONE)
+    return status;
+
+  /* supposed it is ONE if given model is directory */
```

Comment:
Add this handling to documentation in `tensor_filter_common.c` for `model` property as well as to the APIs?

Reply:
IMO this is unnecessary, directory string is only available on NNFW now.

Reply:
ok